src/main/java/bio/terra/service/snapshot/flight/create/SnapshotCreateFlight.java

```diff
@@ -158,10 +161,12 @@ public SnapshotCreateFlight(FlightMap inputParameters, Object applicationContext
         pdaoAclRetryRule);
 
     // Apply the IAM readers to the GCS files
-    addStep(
-        new SnapshotAuthzFileAclStep(
-            dependencyDao, snapshotService, gcsPdao, datasetService, configService),
-        pdaoAclRetryRule);
+    if (platform.isGcp()) {
+      addStep(
+          new SnapshotAuthzFileAclStep(
+              dependencyDao, snapshotService, gcsPdao, datasetService, configService),
+          pdaoAclRetryRule);
+    }
 
     addStep(new SnapshotAuthzBqJobUserStep(snapshotService, resourceService, snapshotName));
```

Comment:
i think this should be under the `platform.isGcp()` as well, i was having issues with it yesterday

Reply:
It will eventually but right now we actually need it since we're still using BQ for azure datasets.  I'm curious about the issues you ran into it with this.  It ran ok for me